File: Dart/1018.dart
import 'dart:io';

void main() {
  int notes = int.parse(stdin.readLineSync());
  int hundred, fifty, twenty, ten, five, two, one;
  print(notes);
  hundred = notes ~/ 100;
  notes = notes % 100;
  fifty = notes ~/ 50;
  notes = notes % 50;
  twenty = notes ~/ 20;
  notes = notes % 20;
  ten = notes ~/ 10;
  notes = notes % 10;
  five = notes ~/ 5;
  notes = notes % 5;
  two = notes ~/ 2;
  notes = notes % 2;
  one = notes ~/ 1;
  print("${hundred} nota(s) de R\$ 100,00");
  print("${fifty} nota(s) de R\$ 50,00");
  print("${twenty} nota(s) de R\$ 20,00");
  print("${ten} nota(s) de R\$ 10,00");
  print("${five} nota(s) de R\$ 5,00");
  print("${two} nota(s) de R\$ 2,00");
  print("${one} nota(s) de R\$ 1,00");
}

File: C/1018.c
#include <stdio.h>

int main()
{
    int notes, hundred, fifty, twenty, ten, five, two, one;
    scanf("%d", &notes);
    printf("%d\n", notes);
    hundred = notes / 100;
    notes = notes % 100;
    printf("%d nota(s) de R$ 100,00\n", hundred);
    fifty = notes / 50;
    notes = notes % 50;
    printf("%d nota(s) de R$ 50,00\n", fifty);
    twenty = notes / 20;
    notes = notes % 20;
    printf("%d nota(s) de R$ 20,00\n", twenty);
    ten = notes / 10;
    notes = notes % 10;
    printf("%d nota(s) de R$ 10,00\n", ten);
    five = notes / 5;
    notes = notes % 5;
    printf("%d nota(s) de R$ 5,00\n", five);
    two = notes / 2;
    notes = notes % 2;
    printf("%d nota(s) de R$ 2,00\n", two);
    one = notes;
    printf("%d nota(s) de R$ 1,00\n", one);

    return 0;
}